test/suite/test_cursor13.py

```diff
@@ -65,16 +65,18 @@ def assert_cursor_cached(self, expect_change):
         if expect_change:
             self.assertGreater(stats[0], self.stat_cursor_cache)
             self.stat_cursor_cache = stats[0]
-        else:
-            self.assertEqual(stats[0], self.stat_cursor_cache)
+        # Stats may change due to background operations in history store cursor
+        #else:
+        #    self.assertEqual(stats[0], self.stat_cursor_cache)
```

Comment:
If we can't assert anything reasonable anymore, should we just get rid of this?

Reply:
Thanks Alex. I think that if we get cursor statistics on the the history store and subtract those from the global stats, then we would have the right values.  But we don't get accurate per-file cursor stats due to WT-6478.  So I'm hoping that in the (near?) future we'll be able to do something useful here.  I'll go and update the description in WT-6478 to make clear that we should both fix the in cursor statistics and then revise these tests.

Reply:
We can also mention the ticket number in a comment here, otherwise, someone who is looking at the test will wonder why the else portions are commented out. eg:

`# WT-6478 has more details on why this portion of test is disabled`